ledger/participant-integration-api/src/test/suite/scala/platform/store/ConversionsSpec.scala

```diff
@@ -14,37 +14,43 @@ class ConversionsSpec extends AsyncWordSpec with Matchers {
     "convert an 'Inconsistent' rejection reason" in {
       val reason = domain.RejectionReason.Inconsistent("This was not very consistent.")
       val converted = reason.toParticipantStateRejectionReason
-      converted should be(state.RejectionReasonV0.Inconsistent("This was not very consistent."))
+      converted.code should be(Status.Code.ABORTED.value())
+      converted.message should be("Inconsistent: This was not very consistent.")
     }
 
     "convert an 'Disputed' rejection reason" in {
       val reason = domain.RejectionReason.Disputed("I dispute that.")
       val converted = reason.toParticipantStateRejectionReason
-      converted should be(state.RejectionReasonV0.Disputed("I dispute that."))
+      converted.code should be(Status.Code.INVALID_ARGUMENT.value())
+      converted.message should be("Disputed: I dispute that.")
     }
 
     "convert an 'OutOfQuota' rejection reason" in {
       val reason = domain.RejectionReason.OutOfQuota("Insert coins to continue.")
       val converted = reason.toParticipantStateRejectionReason
-      converted should be(state.RejectionReasonV0.ResourcesExhausted("Insert coins to continue."))
+      converted.code should be(Status.Code.ABORTED.value())
```

Comment:
Why is this error not a `RESOURCE_EXHAUSTED`?

Reply:
I didn't want to change behavior. I agree that this is probably wrong.

Reply:
Makes sense. Let's double-check the mapping between rejection reasons and gRPC error codes later.